packages/client-core/src/networking/AvatarSpawnSystem.tsx

```diff
@@ -173,9 +182,39 @@ const reactor = () => {
   const sceneEntity = useLoadedSceneEntity(locationSceneURL)
   const gltfLoaded = GLTFComponent.useSceneLoaded(sceneEntity)
 
+  const cameraSettingsQuery = useQuery([CameraSettingsComponent])
```

Comment:
the useQuery here should instead be a `useChildrenWithComponents(sceneEntity, [CameraSettingsComponent])` to ensure we preserve multi-scene functionality